tasks/process_agent.py

```diff
@@ -129,6 +129,7 @@ def build_dev_image(ctx, image=None, push=False, base_image="datadog/agent:lates
 
         ctx.run(f"cp pkg/ebpf/bytecode/build/*.o {docker_context}")
         ctx.run(f"cp pkg/ebpf/bytecode/build/runtime/*.c {docker_context}")
+        ctx.run(f"chmod 0644 {docker_context}/*.o {docker_context}/*.c")
```

Comment:
Would `444` be more appropriate? You don't need to modify the `.o` and `.c` files after the copy, right?

Reply:
We are checking explicitly for `0644` elsewhere. The files are getting copied to a tmp folder, so they don't need to modified after copy.

Reply:
Are you talking about https://github.com/DataDog/datadog-agent/blob/23a514fb1c577b1c5ee4b2df32f1a476f50ccc3e/pkg/ebpf/bytecode/permissions.go#L29 ?
The comment is wrong, we are checking root ownership and that group and global do not have write permissions, so `444` would still pass that check.